locales/fr-FR.json

```diff
@@ -49,8 +49,8 @@
     "confirm": "Confirmer",
     "edit": "Éditer",
     "enter_app": "Entrer dans l'application",
-    "favourite": "Ajouter aux messages aimés",
-    "favourited": "Ajouté aux messages aimés",
+    "favourite": "J'aime",
+    "favourited": "J'aime",
```

Comment:
```suggestion
    "favourited": "Aimé",
```

Reply:
Yet previous choices were made accordingly to popular social media practises, and that one's as much relevant.

Reply:
I didn't check but I assumed the difference between `favourite` and `favourited` is whether the *like* button is active or not, right ? If yes, then using "J'aime" for both situations is accurate in French, isn't it ?

Reply:
I understand but, actually, I think in the case it's when you like the post, you can change the form of the sentence to give the current status.
THe action is "J'aime", and the result is "Aimé", or "Vous aimez"

Reply:
Alright.